src/reference/chat/data.json

```diff
@@ -658,11 +658,20 @@
       }
     },
     "PurgeMessage": {
-      "description": "Sent when a user's messages are purged.",
+      "description": "Sent when a user's messages are purged. Note: The moderator object is only set when a user is purged from in chat. The ban event triggers a purge but the object is not defined.",
```

Comment:
Would it be worth it tho? You already get told the banner from the Ban event; so why would you need it on the Purge event? I understand consistency but in terms of usefulness, I don't see one.